Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/nbclient/client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/asyncio/base_events.py", line 685, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from ngsolve import *
from ngsolve.webgui import Draw
from libmyfe import *
mesh = Mesh(unit_square.GenerateMesh(maxh=0.2, quad_dominated=False))
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [0;32mIn[1], line 3[0m
[1;32m      1[0m [38;5;28;01mfrom[39;00m [38;5;21;01mngsolve[39;00m [38;5;28;01mimport[39;00m [38;5;241m*[39m
[1;32m      2[0m [38;5;28;01mfrom[39;00m [38;5;21;01mngsolve[39;00m[38;5;21;01m.[39;00m[38;5;21;01mwebgui[39;00m [38;5;28;01mimport[39;00m Draw
[0;32m----> 3[0m [38;5;28;01mfrom[39;00m [38;5;21;01mlibmyfe[39;00m [38;5;28;01mimport[39;00m [38;5;241m*[39m
[1;32m      4[0m mesh [38;5;241m=[39m Mesh(unit_square[38;5;241m.[39mGenerateMesh(maxh[38;5;241m=[39m[38;5;241m0.2[39m, quad_dominated[38;5;241m=[39m[38;5;28;01mFalse[39;00m))

[0;31mModuleNotFoundError[0m: No module named 'libmyfe'

